.gitignore

```diff
@@ -1,4 +1,5 @@
 .idea/
 venv/
 venv2/
-_build/
\ No newline at end of file
+_build/
+*~
```

Comment:
I haven't used this pattern before. I'm curious 🐱 could you explain to me what it ignores?

Reply:
When you edit a file then backup files created by emacs (and other editors, but emacs is the culprit in my case). These are similar to the .swp(~) files created by vim. This ignores these backup files so that they don't get checked in to the repo.

Reply:
Could in principle also add:
*.swp
*.swo
to ignore vim-specific files, but I don't know if anyone is using vim for this!

Reply:
Ah cool, I think I do see like docx~ and ppt~ files sometimes when they're open - just didn't make the connection! Thanks for the explanation!